builder/parser/line_parsers.go

```diff
@@ -209,34 +208,28 @@ func parseString(rest string) (*Node, map[string]bool, error) {
 
 // parseJSON converts JSON arrays to an AST.
 func parseJSON(rest string) (*Node, map[string]bool, error) {
-	var (
-		myJson   []interface{}
-		next     = &Node{}
-		orignext = next
-		prevnode = next
-	)
-
+	var myJson []interface{}
```

Comment:
Sorry @LK4D4, I don't think we can win here, unless you have something up your sleeve to tell `json` to only parse arrays...  :wink: